astropy_helpers/sphinx/ext/automodsumm.py

```diff
@@ -119,6 +119,7 @@ class Automodsumm(BaseAutosummary):
     option_spec = dict(Autosummary.option_spec)
     option_spec['functions-only'] = flag
     option_spec['classes-only'] = flag
+    option_spec['no-functions-or-classes'] = flag
```

Comment:
Maybe instead of `no-functions-or-classes`, which is awkward, this could just be called `'global-variables-only'`.